services/app-api/prisma/schema.prisma

```diff
@@ -43,18 +43,33 @@ model State {
 }
 
 model User {
-  id                 String             @id
-  createdAt     DateTime               @default(now())
-  updatedAt     DateTime               @default(now()) @updatedAt
-  givenName          String
-  familyName         String
-  email              String
-  role               Role
-  divisionAssignment Division?
-  stateAssignments   State[]
-  stateCode          String?
-  questions          Question[]
-  responses          QuestionResponse[]
+  id                    String             @id
+  createdAt             DateTime           @default(now())
+  updatedAt             DateTime           @default(now()) @updatedAt
+  givenName             String
+  familyName            String
+  email                 String
+  role                  Role
+  divisionAssignment    Division?
+  stateAssignments      State[]
+  stateCode             String?
+  questions             Question[]
+  responses             QuestionResponse[]
+  audits                UserAudit[]
+  updatedByAuditEntries UserAudit[]        @relation("UpdatedBy")
+}
+
+model UserAudit {
+  id              String      @id
+  createdAt       DateTime    @default(now())
+  updatedAt       DateTime    @default(now()) @updatedAt
```

Comment:
Do we still need `@default(now())` for `updatedAt`? Since this is a new table, we would not have old records without this field. Or am I recalling how this works completely wrong.